vue/src/components/document/list.vue

```diff
@@ -34,56 +34,60 @@ export default
     edit: (doc, $mdMenu) ->
       EventBus.$emit 'initializeDocument', doc, $mdMenu
 
+    deleteDocument: (document) ->
+      EventBus.$emit 'openModal',
+        component: 'ConfirmModal'
+        props:
+          confirm:
+            submit: document.destroy
+            text:
+              title:    'attachments.confirm.title'
```

Comment:
This looks great, except we already have strings for this stuff, lets not add any new ones. I'm trying to reduce the number of strings we have to manage in loomio.

comment_form.attachments.remove_attachment
poll_common_delete_modal.question
poll_common_delete_modal.success

should get you through.

Thank you very much!